packages/web-app-files/src/components/AppBar/AppBar.vue

```diff
@@ -110,6 +111,11 @@ export default defineComponent({
     ViewOptions
   },
   props: {
+    viewModeDefault: {
+      type: String,
+      required: false,
+      default: ''
```

Comment:
We should set `ViewModeConstants.defaultModeName` as `default` here. Otherwise it will use `''` which then overwrites the default defined in the `ViewOptions` component.

Reply:
thanks for bringing it up to my attention again, as I wanted to ask you about that (maybe you remember I suggested setting this when we investigated it at first).